src/ConfigConverter/Commands/ConvertConfigToAsciiDocFormatCommand.php

```diff
@@ -89,10 +90,16 @@ public function __construct()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $this->convertFile(
-            $input->getOption('input-file'),
-            $input->getOption('output-file')
-        );
+		if (!empty($input->getOption('input-file')) and !empty($input->getOption('output-file'))) {
```

Comment:
As this is the second time this line's being used, perhaps refactor it into a utility function.